models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -56,8 +58,8 @@ kyberswap_dex AS (
         ,t.evt_tx_hash                                                                 AS tx_hash
         ,''                                                                            AS trace_address
         ,t.evt_index
-    FROM {{ source('kyber_avalanche_c', 'Elastic_Pool_evt_swap') }} t
-    INNER JOIN {{ source('kyber_avalanche_c', 'Elastic_Factory_evt_PoolCreated') }} p
+    FROM {{ source('kyber_avalanche_c_trades', 'Elastic_Pool_evt_swap') }} t
+    INNER JOIN {{ source('kyber_avalanche_c_trades', 'Elastic_Factory_evt_PoolCreated') }} p
```

Comment:
We can change back the source name so it can match the name in dune V2 engine.
```suggestion
    FROM {{ source('kyber_avalanche_c', 'Elastic_Pool_evt_swap') }} t
    INNER JOIN {{ source('kyber_avalanche_c', 'Elastic_Factory_evt_PoolCreated') }} p
```